cp3pt0-deployment/common/migrate_singleton.sh

```diff
@@ -92,14 +180,30 @@ function restore_ibmlicensing() {
 
 function backup_ibmlicensing() {
 
-    instance=`"${OC}" get IBMLicensing instance -o yaml --ignore-not-found | "${YQ}" '
-        with(.; del(.metadata.creationTimestamp) |
-        del(.metadata.managedFields) |
-        del(.metadata.resourceVersion) |
-        del(.metadata.uid) |
-        del(.status)
-        )
-    ' | sed -e 's/^/    /g'`
+    # for LS + LSR installed in the same cluster, the sender needs to be removed as it is not valid any more after migration
+    local reporterURL=$("${OC}" get IBMLicensing instance  --ignore-not-found -o=jsonpath='{.spec.sender.reporterURL}')
+    if [[ $reporterURL == "https://ibm-license-service-reporter:8080" ]]; then
```

Comment:
Is this the only variant indicating the local Reporter?

Reply:
It should be the only one. @martabereta please comment

Reply:
We've discussed it with @szymon-kowalczyk - this is the only option we are aware of